flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -83,7 +88,8 @@
 
 /** IT cases for all cassandra sinks. */
 @SuppressWarnings("serial")
-@Ignore(value = "Flaky test")
+// NoHostAvailableException is raised by Cassandra client under load while connecting to the cluster
+@RetryOnException(times = 3, exception = NoHostAvailableException.class)
```

Comment:
Why is this needed if we have the retry logic in the setup method? The JIRA reads as if the `NoHostAvailableException` only occurs when the client connects to Cassandra. Can this also happen after the `session` is established?

Reply:
It surprised me also but it is needed because `CassandraCommitter#createResource()` calls `cluster.connect();` and most of `CassandraConnectorITCase` tests rely on `CassandraCommitter`

Reply:
Thanks for the clarification @echauchot.